Assets/LeapMotion/Modules/InteractionEngine/Scripts/InteractionManager.cs

```diff
@@ -830,13 +854,93 @@ public class InteractionManager : MonoBehaviour, IRuntimeGizmoComponent {
         bool shouldIgnore = Physics.GetIgnoreLayerCollision(_templateLayer, i);
         Physics.IgnoreLayerCollision(_interactionLayer, i, shouldIgnore);
         Physics.IgnoreLayerCollision(_interactionNoContactLayer, i, shouldIgnore);
-
-        // Set brush layer to collide with nothing
-        Physics.IgnoreLayerCollision(_contactBoneLayer, i, true);
+        Physics.IgnoreLayerCollision(_contactBoneLayer, i, shouldIgnore);
       }
 
-      //After copy and set we enable the interaction between the brushes and interaction objects
+      // Enable interactions between the contact bones and the interaction layer.
```

Comment:
Part of automatic generation is the guarantee that the contact bone layer collides with the interaction layer.  We should provide a warning in the same way we do with the no-contact/bone relationship.  If the configured contact bone layer does not collide with the interaction layer, we should pop a warning in the inspector.